### PR TITLE
[team_time_0] First pr for hello world

### PR DESCRIPTION
**Pull request by TimE9527**
print a string "hello world"
Author: 1178168173@qq.com

### What?

- brief: test code for first pr
- author: time 

### What changes are included in this PR?

add a program

### Are these changes tested?
image

### CODE REVIEW

File: stl/time/time/src/截图.png
Comment (Light-City) [inline stl/time/time/src/截图.png:1]: 截图.png 

It is best to put the picture in the file, and the name is meaningful

Comment (TimE9527) [reply stl/time/time/src/截图.png:1]: Thanks for checking, I've changed the name of the screenshot and put it in a different folder.

File: stl/time/time/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include <string>
+int main()
+{
```
Comment (martinx) [inline stl/time/time/src/main.cpp:4]: Do not use the new line.

```cpp
int main()
{
```

=>

```cpp
int main() {
```

File: stl/time/time/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    std::string welcome = "Hello World!";
```
Comment (martinx) [inline stl/time/time/src/main.cpp:5]: Indent 2 spaces.

Comment (TimE9527) [reply stl/time/time/src/main.cpp]: Hi, I have changed the code format as requested, can I still get LGTM now?

Comment (TimE9527) [reply stl/time/time/src/main.cpp]: > Indent 2 spaces.
Hi, I have changed the code format as requested, can I still get LGTM now?




> Indent 2 spaces.

Comment (TimE9527) [reply stl/time/time/src/main.cpp]: > Indent 2 spaces.

Hi, I have changed the code format as requested, can I still get LGTM now?

**Comment by martinx** [comment]
LGTM

**Comment by martinx** [comment]
please change

[time_0] => [team_time_0]

thx.

**Comment by TimE9527** [comment]
> please change
> 
> [time_0] => [team_time_0]
> 
> thx.

Sorry, now that I've changed the title, can I get into the WeChat group?

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**